tests/unit/components/Camera/StandardCamera.test.js

```diff
@@ -0,0 +1,52 @@
+import { fireEvent, render, screen } from "@testing-library/react-native";
+import StandardCamera from "components/Camera/StandardCamera";
+import { ObsEditContext } from "providers/contexts";
+import React from "react";
+
+const mockedNavigate = jest.fn();
+
+jest.mock( "@react-navigation/native", () => {
+  const actualNav = jest.requireActual( "@react-navigation/native" );
+  return {
+    ...actualNav,
+    useNavigation: () => ( {
+      navigate: mockedNavigate
+    } ),
+    useRoute: () => ( {} )
+  };
+} );
+
+const mockValue = {
+  addCameraPhotosToCurrentObservation: jest.fn(),
+  allObsPhotoUris: [],
+  cameraPreviewUris: []
+};
+
+const renderStandardCamera = () => render(
+  <ObsEditContext.Provider value={mockValue}>
+    <StandardCamera />
+  </ObsEditContext.Provider>
+);
+
+describe( "StandardCamera", ( ) => {
+  test( "should first render with flash disabled", async () => {
+    renderStandardCamera();
+
+    await screen.findByLabelText(
+      "The flash is turned off. Press here to enable it."
+    );
+  } );
+
+  test( "should change to flash enabled on button press", async () => {
+    renderStandardCamera();
+
+    const flashButton = await screen.findByLabelText(
+      "The flash is turned off. Press here to enable it."
+    );
+    fireEvent.press( flashButton );
+
+    await screen.findByLabelText(
```

Comment:
For my own education, what's the advantage of using `screen` over the finder methods returned by `render`?

Reply:
I was following this page by the creator of react-testing-library: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library
"The benefit of using screen is you no longer need to keep the render call destructure up-to-date as you add/remove the queries you need. You only need to type screen. and let your editor's magic autocomplete take care of the rest." And for me also, that I don't have to destructure the render return in every test.